app/javascript/components/upload/ExpandableFileForm.jsx

```diff
@@ -135,6 +135,12 @@ export function SaveDeleteButtons({
     </Button>
   }
 
+  let deleteText
+  if (file.remote_location) {
+    deleteText = ' remain in the bucket because you provided a remote path to an existing file.'
+  } else {
+    deleteText = ' be removed from the bucket.'
```

Comment:
No space needed:

```suggestion
    deleteText = 'remain in the bucket because you provided a remote path to an existing file.'
  } else {
    deleteText = 'be removed from the bucket.'
```